getVersionFromTag.js

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env node
+
+'use strict';
+
+const index = require('./index');
+const nomnom = require('nomnom');
+const opts = nomnom
+    .option('name', {
+        required: true,
+        abbr: 'n',
+        help: 'Template name'
+    })
+    .option('tag', {
+        abbr: 't',
+        required: true,
+        help: 'Tag name'
+    })
+    .parse();
+
+return index.getVersionFromTag({
+    name: opts.name,
+    tag: opts.tag
+})
+    .then((result) => {
+        console.log(`Successfully fetched version ${result.version} from ${opts.tag}`);
```

Comment:
How about

```js
console.log(`${opts.tag} tags ${opts.name}@${result.version}`);
```

as the message?

Reply:
I find `Successfully fetched X from Y` seems like a more appropriate message when attempting to fetch a resource from say, a database, as opposed to simply checking what version the template tag points to.